src/containers/Home.js

```diff
@@ -12,9 +11,9 @@ class Home extends Component {
    */
   render() {
     return (
-      <div>
-        <h2>Welcome to Authors Haven</h2>
-      </div>
+      <Fragment>
```

Comment:
The fragment tag from React is meant to be used to wrap sibling HTML markup kindly refactor the above code.  Great work done so far

Reply:
@gbols, I have not asked for review on this PR, it's still a work in progress. I have done all the refactoring, waiting to complete my test before I push.

Reply:
@gbols. Could you please throw a little color in what you said. For instance, give a code example of how a `Fragment` should be used.